server/msteams/client.go

```diff
@@ -126,6 +126,15 @@ type Activity struct {
 	LifecycleEvent                 string
 	SubscriptionExpirationDateTime time.Time
 	SubscriptionID                 string
+	EncryptedContent               EncryptedContent
```

Comment:
My way of thinking was, using pointer here makes it harder to handle later. The zero value of that structure is an empty `Data` field anyway, so I can check that. If there is no `Data` there is nothing to decrypt, so that is good enough for me, and I don't need to deal with the nil checks. In this case is not a big deal to manage the nil checks, because it is very localized, and probably is more semantically correct. So... I think I going to change it to your proposal.